Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -812,3 +812,36 @@ def toAlgAut : G →* A ≃ₐ[R] A where
 end
 
 end MulSemiringAction
+
+namespace AlgEquiv
+section CompHom
+/--
+Define an equivalence of algebras in the following commutative diagram:
+A₁ <------> A₂
+/\         /\
+|           |
+|           |
+S --------> R
```

Comment:
Can you explain the relation to `Algebra.compHom` here? Using non-canonical instances like `compHom` is quite unusual, so it's important to draw attention to it when it happens; especially since the docs wont show the `letI`s!

Reply:
@yhtq This question is still open. Could you please answer it?